client/source/js/modules/analysis/optimization-ctrl.js

```diff
@@ -9,6 +9,7 @@ define([
 
       $scope.meta = meta;
       $scope.types = angular.copy(CONFIG.GRAPH_TYPES);
+      $scope.optimizationStatus = false;
```

Comment:
As an alternative you could use an object to describe state & related meta data like the text. Here an example:

    var status = {
      NOT_RUNNING: { text: "Optimization is running", isActive: false },
      RUNNING: { text: "", isActive: true },
      REQUESTED_TO_STOP : { text:"Optimization is requested to stop", isActive: true }
    };

    // assign the message
    $scope.optimizationStatus = status.NOT_RUNNING;

    // check if it's running in the template
    ng-disabled="optimizationStatus.isActive"

    // message in the template
    {{ optimizationStatus.text }}